app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -65,21 +70,20 @@ def parse_vm(vm)
     cpus          = vm.cpu
     memory_mb     = vm.memory
     disk_capacity = vm.hard_disks.inject(0) { |sum, x| sum + x.values[0] } * 1.megabyte
-
-    vm_disks = vm.disks.all
-
-    disks = vm_disks.select { |d| d.description == "Hard disk" }.map do |disk|
-      parent = vm_disks.find { |d| d.id == disk.parent }
-
-      {
-        :device_name     => disk.name,
-        :device_type     => "disk",
-        :controller_type => parent.description,
-        :size            => disk.capacity * 1.megabyte,
-        :location        => "#{vm.id}-#{disk.id}",
-        :filename        => "#{vm.id}-#{disk.id}",
-      }
-    end
+    vm_disks      = vm.disks.all
+    disks         = vm_disks.select { |d| d.description == "Hard disk" }.map do |disk|
+                      parent = vm_disks.find { |d| d.id == disk.parent }
+                      {
+                        :device_name     => disk.name,
+                        :device_type     => "disk",
+                        :controller_type => parent.description,
+                        :size            => disk.capacity * 1.megabyte,
+                        :location        => "#{vm.id}-#{disk.id}",
+                        :filename        => "#{vm.id}-#{disk.id}",
+                      }
+                    end
```

Comment:
I'd leave this block alone for this PR, if you want to line the `=` up for `vapp_uid` and `stack` you can move them up beneath `memory_mb` or `disk_capacity`.